R/calculate_protein_abundance.R

```diff
@@ -127,7 +128,7 @@ calculate_protein_abundance <- function(data,
     dplyr::group_by({{ protein_id }}, {{ sample }}) %>%
     dplyr::mutate(n_peptides = dplyr::n_distinct(!!rlang::ensym(peptide))) %>%
     dplyr::filter(.data$n_peptides > 2) %>%
```

Comment:
Why are you using `.data$n_peptides` in line 130 and `"n_peptides"` in line 131? (Not a rhetorical question i really don`t know it)

Reply:
So when coding with tidyverse packages one would refer to column names with `.data$n_peptides` instead of just with `n_peptides`. When you run checks on the package it would otherwise complain that there is no binding to global variable `n_peptides`, because it thinks that it is actually a variable. 

Just of note, as you know I guess you refer to arguments of the functions, which contain column names with `{{}}`, but I guess that is clear.

So recently they have changed that one should refer to columns with `"` in functions that use "tidy selections" arguments while you still keep `.data$` for the "data masking" arguments. So I started at one point changing all the functions because of this... I am sure I still missed quite a few of these cases. So over time the warnings in the checks for all functions will disappear I hope.

Here is also a forum discussion about the subject and it links to the blog post that describes how to refer to things properly: https://community.rstudio.com/t/use-of-data-in-tidyselect-expressions-is-now-deprecated/150092/2